book/src/coding-conventions.md

```diff
@@ -70,3 +70,17 @@ TODO: how to name error types/error enum variants (`CannotDoSth` vs `DoingSth`)?
 ## Dependencies
 
 If the same dependency is used in multiples crates within the workspace, that dependency SHOULD be specified in the *workspace*'s `Cargo.toml` file and workspace crates should import them from there.
+
+## Adding a new workspace crate, exposed by `riot-rs`
+
+To add a new workspace crate re-exported by `riot-rs`, follow these steps:
+
+1. Create the new crate's directory in `src/`.
+1. Run `cargo init --lib` in that directory.
+1. Move the `[lints]` section in its `Cargo.toml` above the `[dependencies]` section.
```

Comment:
This and the next need to be more specific I think, I wouldn't know which to move/copy